WebRoot/public/login.jsp

```diff
@@ -587,12 +587,15 @@ if (application.getInitParameter("offlineMode") != null) {
                                 <input type="submit" class="ZLoginButton DwtButton" value="<fmt:message key="login"/>" />
                                 </td>
                                 </tr>
+				<c:set var="resetPasswordFeatureStatus" value="${domainInfo.attrs.zimbraFeatureResetPasswordStatus}"/>
+				<c:if test="${resetPasswordFeatureStatus eq 'enabled'}">
```

Comment:
no need to set a variable for this, we can do inline
`<c:if test="${domainInfo.attrs.zimbraFeatureResetPasswordStatus eq 'enabled'}">`